### PR TITLE
feat: rework rules catalog page and add correlation

### PR DESCRIPTION
**Pull request by gaelmuller**
Update the Rules Catalog page to:

* Add screenshots with the new theme
* Make some sections clearer
* Remove the section about STIX which is now deprecated
* Add Sigma Correlation

### CODE REVIEW

File: docs/operation_center/rules_catalog.md [hunk 1]
```diff
@@ -1,336 +1,441 @@
...
+
+```yaml
 detection:
   selection:
-    event.type: allowed
-    destination.ip:
-    - 1.2.3.4
-    - 4.3.2.1
+    - source.ip: 6.6.6.6
+    - detection.ip: 6.6.6.6
```
Comment (Thibault-MAL) [inline docs/operation_center/rules_catalog.md:189]: Hello,
Why have you chosen `detection.ip` and not `destination.ip:`  ? :thinking:

Comment (gaelmuller) [inline docs/operation_center/rules_catalog.md:189]:
```suggestion
    - destination.ip: 6.6.6.6
```

File: docs/operation_center/rules_catalog.md [hunk 2]
```diff
@@ -1,336 +1,441 @@
...
+    event.category: authentication
+    event.outcome: success
+  condition: selection
+---
+name: new_network_connection
+detection:
+  selection:
+    event.category: network
+    event.type: connection
+    event.ouctome: success
```
Comment (gaelmuller) [inline docs/operation_center/rules_catalog.md:421]:
```suggestion
    event.outcome: success
```

File: docs/operation_center/rules_catalog.md (hunk 1 above)
Comment (gaelmuller) [reply docs/operation_center/rules_catalog.md:189]: Nice catch, thx :+1:

**Review by CharlesLR-sekoia — APPROVED** [review]

**Pull request merged**